test/git.cwd-owner.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+const t = require('tap')
+
+process.getuid = () => 0
+process.getgid = () => 0
```

Comment:
question: wouldn't this pollute the global `process` and have unintended effects in possible future tests? I'm usually very defensive about it and try to use the `setup` along with `t.teardown` when mocking global stuff but then there might be some **tap** magic that I'm not aware of 🤔

Reply:
It'll only pollute the process for this one test file, but each test file is fully isolated in a different process so it's fine. 

It'd only be a problem if there were multiple tests in the same file.

Reply:
> each test file is fully isolated in a different process so it's fine.

oooh, that's the part I was missing then! :D thanks!